### PR TITLE
Sprikipedia 

### PR DESCRIPTION
**Pull request by itsimpeccable**
I did moderate changes to the sprikipedia, adding more team members to team page, adding a sprikipedia redirection URL to the homepage, and I made Philosophy and Acknowledgements page to the one page named "About Sprig".

### CODE REVIEW

File: src/components/navbar-main.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default function MainNavbar(props: MainNavbarProps) {
 				) : null}
 				<li><a href='/gallery'>Gallery</a></li>
 				<li><a href='/get'>Get a Sprig</a></li>
+				<li><a href='/sprikipedia'>Sprikipedia</a></li>
```
Comment (LucasHT22) [inline src/components/navbar-main.tsx:29]: Sprikipedia isn't ship'ed yet! So we won't add it to the main navbar for now

File: src/pages/sprikipedia/getting-started/about-sprig.astro [hunk 2]
```diff
@@ -31,4 +31,4 @@ const data = (
       name="CodeMirror"
       description=" " />
 </div>
-</Categories>
```
Comment (LucasHT22) [inline src/pages/sprikipedia/getting-started/about-sprig.astro:34]: Disagree with the rename, getting-started folder follow README.md, and in README.md is https://github.com/hackclub/sprig#acknowledgements

File: src/pages/sprikipedia/getting-started/philosophy.astro [hunk 3]
```diff
@@ -1,14 +0,0 @@
...
-
-const data = (
-  (await Astro.glob(`/src/translations/*.json`)).find(
-    (translation) =>
-      translation.default.lang == (Astro.url.searchParams.get("lang") || "en_US")
-  ) || {}
-).default;
----
-<Categories name={data["start.categories.name.philosophy"]} title={data["start.categories.title.philosophy"]} >
-<p>People learn best when they make things that they care about, which they can then share with others. This type of learning philosophy is called constructionism, and Sprig is a type of microworld. A microworld is an environment where you can discover programming by using it to express yourself.</p>
```
Comment (LucasHT22) [inline src/pages/sprikipedia/getting-started/philosophy.astro:13]: Why delete it? 👀

File: src/pages/sprikipedia/team/index.astro [hunk 4]
```diff
@@ -14,9 +14,19 @@ const data = (
 <Categories name={data["team.categories.name"]} title={data["team.categories.title"]} >
     <p>{data["team.categories.about"]}</p>
```
Comment (LucasHT22) [inline src/pages/sprikipedia/team/index.astro:15]: I appreciate your changes but I'm already preparing for spriki29 with reference to https://github.com/hackclub/sprig/blob/main/RESPONSIBILITIES.md :)

File: games/LabyrinthRedux.js [hunk 5]
```diff
@@ -0,0 +1,288 @@
+/*
```
Comment (LucasHT22) [inline games/LabyrinthRedux.js:1]: Please make another PR for your game:)

**Review by LucasHT22 — CHANGES_REQUESTED** [review]
Hey! Really thank you for this PR! Make some comments, please don't get me wrong, I'm just being careful with this:)

**Comment by Sheepy3** [comment]
Hi, is this PR still active?

**Comment by grymmy** [comment]
I am closing this PR only due to its old age, and thus it seems unlikely it will land.  @LucasHT22 feel free, absolutely, to reopen this PR if you desire to push it towards landing to mainline.

**Comment by itsimpeccable** [comment]
I was not active about sprikipedia and yeah graham, you should've closed
this already.

**Pull request closed without merging**